base/sysimg.jl

```diff
@@ -115,9 +115,9 @@ include("intfuncs.jl")
 include("nullable.jl")
 
 # I/O
+include("show.jl")
 include("task.jl")
 include("lock.jl")
-include("show.jl")
```

Comment:
this is the kind of thing that's worth mentioning at least briefly in the commit message